src/client/src/pages/home/HomeView.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import COLORS from '../../assets/colors';
+import FONTS from '../../assets/fonts';
+
+function App({ children }: any) {
+  return (
+    <div
+      style={{
+        backgroundColor: COLORS.header,
+        width: '100px',
+        height: '100px',
+        fontFamily: FONTS.headerFont,
+        fontWeight: FONTS.headerWeight,
+      }}
```

Comment:
Please use styled components!

Reply:
Actually since you are using this a lot, can you make a styled component and put it in the common components folder and then import it into the files. Thank you!

Reply:
This is just a placeholder for all the pages (just so that something is displayed) -- should I still fix this?